community/mei-by-laws.md

```diff
@@ -73,41 +77,51 @@ The Board is self-organizing; that is, there will be no formal regulations on ho
 
-The elected members of the Board each are elected for a three year term. Terms on the Board are staggered so that three positions are open for election each year. Terms of office begin January 1 at 2 pm UTC.
+### 5.2 Meetings
 
-Exception to annual elections and term lengths will be made for the initial election of the Board (see Initial election of the Board) and in case a Board member resigns.
+The Board holds regular, virtual or in-person meetings, e.g. approximately every three months. As schedules and funding permit, the Board meets in person at least once per year. For every meeting, an agenda shall be published online prior to the meeting and minutes of the meeting shall be provided in a timely manner following the meeting. Depending on the availability of funding, the members of the Board may be refunded travel costs for this meeting.
+
+### 5.3 Terms & Elections
 
-Board members may resign from their commitments at any time but are expected to give adequate notice. After accepting the resignation, the Board may decide on how to proceed concerning the newly vacant position. Options should include but are not limited to holding a special election, appointing a replacement, or leaving the position vacant.
+The elected members of the Board each are elected for a three year term. Terms on the Board are staggered so that three positions are open for election each year. Terms of office begin January 1 at 2 pm UTC.
 
-In case of a special election to fill a vacancy between regular elections the newly elected Board member will serve for the remainder of the term of the vacant position.
+Exception to annual elections and term lengths will be made for the initial election of the Board and in case a Board member resigns.
 
 #### 5.3.1 Eligibility
 
 In order to be available for election to the Board, candidates must hold Personal Membership in MEI. Individuals who have previously served on the Board will be eligible for re-appointment.
 
 #### 5.3.2 Election procedure
 
-In order to determine a range of candidates, a four-week nomination phase will precede the elections, in which self-nominations are permitted. Individuals will be informed of their nomination and asked to confirm their willingness to serve on the Board determined. Elections will be held in a timely manner after determination of a final list of candidates. A ranked voting system without minimum voter turnout will be applied.
+In order to determine a range of candidates, a two-week nomination phase will precede the elections, in which self-nominations are permitted. Individuals will be informed of their nomination and asked to confirm their willingness to serve on the Board determined. Elections will be held in a timely manner after determination of a final list of candidates. Elections should be open for at least two weeks. Elections should close before the start of the terms of office of the elected candidates.
```

Comment:
Maybe use "shall" instead of "should" in two places in this paragraph.
"Should" is a suggestion word. "Shall" is a command word.  Do you want to give the Elections Officers and Board the flexibility to have shorter election times, or elections after the start of terms of office? Or do you want them to always have two+ week times, closed before the start of the terms of office? It is a policy choice.

Reply:
Thank you. Changed to "shall" in both places.